node/subsystem/src/util.rs

```diff
@@ -663,6 +663,9 @@ where
 					}
 				}
 			}
+			Ok(Signal(BlockFinalized(_))) => {
+				return false;
+			}
```

Comment:
```suggestion
			Ok(Signal(BlockFinalized(_))) => {}
```

We can just let the operation fall through in this case.